src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FinalLocalVariableCheck.java

```diff
@@ -129,6 +136,13 @@ public class FinalLocalVariableCheck extends AbstractCheck {
      */
     private boolean validateEnhancedForLoopVariable;
 
+    /**
+     * Control whether to check
+     * <a href="https://docs.oracle.com/javase/specs/jls/se21/preview/specs/unnamed-jls.html">
+     * unnamed variables </a>.
+     */
```

Comment:
JLS doesn't have a separate section for this I just went to (https://docs.oracle.com/javase/specs/index.html) and copied the link of the last review for this feature is this ok?

Reply:
Is there some way we can use a non-preview JLS? Nothing we are doing is preview only, right?

Reply:
> Is there some way we can use a non-preview JLS?

I don't think so. All items listed [here](https://docs.oracle.com/javase/specs/) are previews. we can use a non-perview from [here](https://docs.oracle.com/javase/specs/jls/se22/html/index.html) if the feature has a separate new section